qemu/tests/blockdev_commit_non_existed_base.py

```diff
@@ -0,0 +1,43 @@
+import logging
+
+from virttest.qemu_monitor import QMPCmdError
+
+from provider import backup_utils
+from provider.blockdev_commit_base import BlockDevCommitTest
+
+
+class BlockdevCommitNonExistedBase(BlockDevCommitTest):
+
+    def commit_snapshots(self):
+        device = self.params.get("device_tag")
+        device_params = self.params.object_params(device)
+        snapshot_tags = device_params["snapshot_tags"].split()
+        self.device_node = self.get_node_name(device)
+        options = ["base-node", "top-node", "speed"]
+        arguments = self.params.copy_from_keys(options)
+        arguments["base-node"] = 'sn0'
```

Comment:
suggest that setting sn0 in cfg, e.g. none_existed_sn = sn0, then use '${none_existed_sn}' in qmp_error_msg definition.